### ISSUE TITLE
Area (hub) pages accessibility issues

### ISSUE THREAD

**Issue by petem-ons** [ONSvisual/local-statistics#22]
1. "Explore data for ..." on area hub page should be an <h2> tag.
2. Markup for nav map. Currently there's an aria-described-by on a div. (Should say "interactive map, use the accompanying links")
3. Tab panel regions (for child areas) don't seem to have the right markup. (check W3 for best practice example)
4. Make sure focus goes to the right place when "show more" is clicked.
5. There seems to be a bug that "show more" shows even when all content is displayed.
6. For area navigation, do we need a description of HOW to navigate?
7. Area hierarchy links should be <ol> not spans.
8. Card images should have a NULL alt text. alt=""
9. Search boxes should have a proper programmatically associated name. That should be "find/search for another area", not "type a place name or postcode". (use both <label for=""> and aria-described-by="" in combination).
10. Search dropdown list getting cut off when it's inside a card (check card overflow).
